### PR TITLE
A few extra improvements to readability.

### PR DESCRIPTION
**Pull request by arfon**

### CODE REVIEW

File: joss/paper.md [hunk 1]
```diff
@@ -51,20 +51,20 @@ _tfaip_ resolves the following different aspects in an elegant and robust way.
...
-This is particularly sensible if the scenarios and thus their code basis becomes larger since it leads to a clean, structured, modularized, and readable code preventing bad code style and facilitates maintenance.
+In comparison to other frameworks such as Tensorflow, _tfaip_ requires users to implement their scenarios in an object-oriented way and encourages them to annotate their code with types.
+This is particularly advantageous for larger, more complex scenarios where the codebase grows accordingly. Using _tfaip_ leads to a clean, structured, modularized, and readable code limiting poor coding styles and facilitating maintenance.
 In practice, each scenario is created by implementing predefined interfaces (e.g., loss-function or the graph construction).
 
 During research and development, a tedious step is data preparation which often comprises the conversion of data into the format required by the framework.
-The Tensorflow-backed of _tfaip_ allows integrating Python code in the data pipeline which is however not run (truly) in parallel by multiple processes and results quite often in a bottleneck.
```
Comment (ChWick) [inline joss/paper.md:65]: I think the "of" is mandatory here, since `tfaip` is no Tensorflow backend, but vice-versa.

File: joss/paper.md [hunk 2]
```diff
@@ -51,20 +51,20 @@ _tfaip_ resolves the following different aspects in an elegant and robust way.
...
-To speed-up Tensorflow, a user has to transform Python into Tensorflow operations which is laborious, partly even impossible, and complicates debugging.
-_tfaip_ tackles this issue by providing a sophisticated pipeline setup based on so-called data processors which apply simple transformation operations in pure Python code and are automatically executed in parallel.
+The Tensorflow-backed _tfaip_ allows integrating Python code in the data pipeline which is however not run (truly) in parallel by multiple processes and results quite often in a bottleneck.
+To speed-up Tensorflow, a user has to transform Python into Tensorflow operations which is laborious, and sometimes even impossible, and complicates debugging.
+_tfaip_ tackles this issue by providing a sophisticated pipeline setup based of 'data processors' which apply simple transformation operations in pure Python code and are automatically executed in parallel.
```
Comment (ChWick) [inline joss/paper.md:67]: I'd suggest changing this to double quotes and "based on":

> pipeline setup based on "data processors"

**Review by ChWick — COMMENTED** [review]
Thank you so much for the improvements. I have 2 questions left (see comments)

File: joss/paper.md (hunk 1 above)
Comment (arfon) [reply joss/paper.md:65]: Yes, Tensorflow-backed _tfaip_ means '_tfaip_ supported/backed by Tensorflow'.

Leaving 'of' in this sentence doesn't make sense.

File: joss/paper.md (hunk 2 above)
Comment (arfon) [reply joss/paper.md:67]:
```suggestion
_tfaip_ tackles this issue by providing a sophisticated pipeline setup based on "data processors" which apply simple transformation operations in pure Python code and are automatically executed in parallel.
```

**Comment by arfon** [comment]
OK, I think this is good to merge now @ChWick

File: joss/paper.md (hunk 1 above)
Comment (ChWick) [reply joss/paper.md:65]: You are absolutely right, my mistake, I missed the missing an "n":
* Either `Tensorflow-backed _tfaip_` or
* `Tensorflow-backeNd of _tfaip_`.

Sorry.

**Review by ChWick — APPROVED** [review]

**Pull request merged**